tests/typecheck/managers/querysets/test_from_queryset.yml

```diff
@@ -310,6 +310,26 @@
                 class MyModel(models.Model):
                     objects = NewManager()
 
+-   case: maybe_weird_bug
```

Comment:
It passes!

But, the title of this PR is "Add failing test".
I am confused :)

Reply:
This is related to the `error: Name "Sequence" is not defined` issue @sobolevn. I tried to reproduce it in #1021 as well, but for some reason it doesn't reproduce on CI. It reproduces locally and I've created a separate project here where it reproduces as well: https://github.com/ljodal/django-stubs-regression-test The regression was introduced in #991

Reply:
> I am confused :)

I am also confused. :~)